### PR TITLE
Update se.m3u

### PR DESCRIPTION
**Pull request by freearhey**
Added link from https://github.com/iptv-org/iptv/issues/1951#issuecomment-830266328 and https://github.com/iptv-org/iptv/pull/3056#issuecomment-857192767

### CODE REVIEW

File: channels/se.m3u [hunk 1]
```diff
@@ -49,3 +49,5 @@ http://195.189.105.158:9050/134/index.m3u8
 https://edg01-prd-se-dcs.solidtango.com/edge/_lo9yf4l5_/lo9yf4l5/playlist.m3u8
 #EXTINF:-1 tvg-id="OppnaKanalen.se" tvg-name="Öppna Kanalen" tvg-country="SE" tvg-language="Swedish" tvg-logo="https://i.imgur.com/O1wCZTZ.jpg" group-title="Local",Öppna Kanalen (540p)
 http://83.140.64.214/edge/_451iw2h_/451iw2h/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV10
```
Comment (LaneSh4d0w) [inline channels/se.m3u:52]: Not quite sure that's the correct country. I see CBN News
vlcsnap-2021-06-07-21h19m10s277

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]

File: channels/se.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/se.m3u:52]: image
It was reffering CBN News

Comment (Dum4G) [reply channels/se.m3u:52]: @LaneSh4d0w

Comment (LaneSh4d0w) [reply channels/se.m3u:52]: https://en.wikipedia.org/wiki/TV10_(Sweden)

TV10 doesn't have this logo, it's this one : https://upload.wikimedia.org/wikipedia/en/7/7a/TV10_logo_2010_better.JPG

Comment (Dum4G) [reply channels/se.m3u:52]: Found one, religious channel @LaneSh4d0w  https://www.lyngsat.com/tvchannels/se/Kanal-10-Sverige.html
https://sv.wikipedia.org/wiki/Kanal_10

**Comment by Dum4G** [comment]
Kanal 10 Asia, 540p
https://cdn-kanal10.crossnet.net/kanal10/ngrp:kanal10asia_all/chunklist_w522739009_b2128000.m3u8

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**